Documentation for Oracle Debezium connector (#12213)
apache/pulsar: site2/docs/io-debezium-source.md

    ```
   
## Example of Oracle

### Packaging

> Oracle connector does not include Oracle JDBC driver and the end user will need to package it with the connector.

Reviewer: ```suggestion
Oracle connector does not include Oracle JDBC driver and you need to package it with the connector.
```

> Major reasons for not including the drivers are the variety of versions and Oracle licensing. It is recommended using the driver provided with your Oracle DB installation, or you can [download](https://www.oracle.com/database/technologies/appdev/jdbc.html) one.

Reviewer: ```suggestion
Major reasons for not including the drivers are the variety of versions and Oracle licensing. It is recommended to use the driver provided with your Oracle DB installation, or you can [download](https://www.oracle.com/database/technologies/appdev/jdbc.html) one.
```

Integration test have an [example](https://github.com/apache/pulsar/blob/e2bc52d40450fa00af258c4432a5b71d50a5c6e0/tests/docker-images/latest-version-image/Dockerfile#L110-L122) of packaging the driver into the connector nar file. 

### Configuration

Debezium [requires](https://debezium.io/documentation/reference/1.5/connectors/oracle.html#oracle-overview) Oracle DB with LogMiner or XStream API enabled. 
Supported options and steps for enabling them vary from version to version of Oracle DB.
Steps outlined in the [documentation](https://debezium.io/documentation/reference/1.5/connectors/oracle.html#oracle-overview) and used in the [integration test](https://github.com/apache/pulsar/blob/master/tests/integration/src/test/java/org/apache/pulsar/tests/integration/io/sources/debezium/DebeziumOracleDbSourceTester.java) may or may not work for the version and edition of Oracle DB you are using.

> Please refer to the documentation for Oracle DB as needed.

Reviewer: Can you add a reference link here?

Author: done

> Similarly to other connectors, you can use JSON or YAMl to configure the connector.

Reviewer: Like other connector docs, can you give a JSON example?

Author: done

Using yaml as an example, you can create a debezium-oracle-source-config.yaml file like:

```yaml
tenant: "public"
namespace: "default"
name: "debezium-oracle-source"
topicName: "debezium-oracle-topic"
parallelism: 1

className: "org.apache.pulsar.io.debezium.oracle.DebeziumOracleSource"
database.dbname: "XE"

configs:
    database.hostname: "localhost"
    database.port: "1521"
    database.user: "dbzuser"
    database.password: "dbz"
    database.dbname: "XE"
    database.server.name: "XE"
    schema.exclude.list: "system,dbzuser"
    snapshot.mode: "initial"
    topic.namespace: "public/default"
    task.class: "io.debezium.connector.oracle.OracleConnectorTask"
    value.converter: "org.apache.kafka.connect.json.JsonConverter"
    key.converter: "org.apache.kafka.connect.json.JsonConverter"
    typeClassName: "org.apache.pulsar.common.schema.KeyValue"
    database.history: "org.apache.pulsar.io.debezium.PulsarDatabaseHistory"
    database.tcpKeepAlive: "true"
    decimal.handling.mode: "double"
    database.history.pulsar.topic: "debezium-oracle-source-history-topic"
    database.history.pulsar.service.url: "pulsar://127.0.0.1:6650"
```

> Full list of configuration properties supported by Debezium is available in the [documentation](https://debezium.io/documentation/reference/1.5/connectors/oracle.html#oracle-connector-properties).

Reviewer: ```suggestion
For the full list of configuration properties supported by Debezium, see [Debezium Connector for Oracle](https://debezium.io/documentation/reference/1.5/connectors/oracle.html#oracle-connector-properties).
```
